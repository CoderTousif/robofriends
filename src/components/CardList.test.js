import React from 'react';
import CardList from './CardList';
import { shallow } from 'enzyme';

it('should render cardlist component', () => {
    // expect(shallow(<CardList />).length).toEqual(1);
    const mockRobots = [
        {
            id: 1,
            name: 'Leanne Graham',
            username: 'Bret',
            email: 'Sincere@april.biz',
        },
        {
            id: 2,
            name: 'Ervin Howell',
            username: 'Antonette',
            email: 'Shanna@melissa.tv',
        },
    ];

    expect(shallow(<CardList arr={mockRobots} />)).toMatchSnapshot();
});
