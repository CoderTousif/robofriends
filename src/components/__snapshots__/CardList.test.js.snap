// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render cardlist component 1`] = `ShallowWrapper {}`;
